boards/common/samd21-arduino-bootloader/Makefile.include

```diff
@@ -0,0 +1,31 @@
+# setup the flash tool used
+# Bossa is the default programmer
+PROGRAMMER ?= bossa
+
+ifeq ($(PROGRAMMER),bossa)
+  # by default, we use BOSSA to flash this board and take into account the
+  # preinstalled Arduino bootloader. ROM_OFFSET skips the space taken by
+  # such bootloader.
+  ROM_OFFSET ?= 0x2000
+  BOSSA_ARDUINO_PREFLASH = yes
+  PREFLASH_DELAY = 1
+
+  ifneq (,$(filter reset flash flash-only, $(MAKECMDGOALS)))
+    # By default, add 2 seconds delay before opening terminal: this is required
+    # when opening the terminal right after flashing. In this case, the stdio
+    # over USB needs some time after reset before being ready.
+    TERM_DELAY ?= 2
+    TERMDEPS += term-delay
+  endif
+
+  include $(RIOTMAKE)/tools/bossa.inc.mk
+endif
+
+term-delay:
+	sleep $(TERM_DELAY)
```

Comment:
This is not very reliable from my experience, many times this will take longer. It could be nice to poll the terminal until ready for this `BOARD`. But this is out of scope, we can leave it for a followup PR. :)